packages/react-components/.storybook/main.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const path = require('path');
+
+const rootMain = require('../../../.storybook/main');
+
+module.exports = /** @type {Pick<import('../../../.storybook/main').StorybookConfig,'addons'|'stories'|'webpackFinal'>} */ ({
+  stories: [...rootMain.stories, '../src/**/*.stories.mdx', '../src/**/*.stories.@(ts|tsx)', ...getVnextStories()],
+  addons: [...rootMain.addons],
+  webpackFinal: (config, options) => {
+    const localConfig = { ...rootMain.webpackFinal(config, options) };
+
+    return localConfig;
+  },
+});
+
+function getVnextStories() {
```

Comment:
custom logic to parse for component stories that are part of `react-components` suite